source/features/follow-file-renames.tsx

```diff
@@ -69,5 +69,9 @@ features.add({
 	include: [
 		pageDetect.isRepoCommitList
 	],
+	exclude: [
+		// Probably looking at the base /commits/<branch> page, not a subfolder or file.
+		() => !select.exists('.breadcrumb')
```

Comment:
Add to page-detect?
`isCommitHistoryPage` => isRepoCommitList && exists('.breadcrumb')

Reply:
Too specific IMHO